src/util/sqlhelper.cpp

```diff
@@ -301,6 +304,12 @@ namespace hsql {
     inprint("CreateStatement", numIndent);
     inprint(stmt->tableName, numIndent + 1);
     if (stmt->filePath) inprint(stmt->filePath, numIndent + 1);
+    if (stmt->columns) {
```

Comment:
Since improving the `printCreateStatementInfo` does not only address the `print select distinct` issue (#19), I would change the PR description respectively. Accordingly, the title of the PR should be descriptive and reflect the content of the PR.

In addition, if we parse, for example, `CREATE TABLE t1 (c1 INTEGER, c2 INTEGER);`, the implementation of this PR results in the following print output:
```
CreateStatement
	t1
		c1
			INT
		c2
			INT
```

Something like the following would be more descriptive, IMHO:
```
CreateStatement
	Type:
		Table
	Name:
		t1
	Columns:
		c1
			INT
		c2
			INT
```

Since a `CreateStatement` can have different `CreateType`s, adding the type to the print output would be beneficial.

Reply:
I just saw #179, so I think the changes in `printCreateStatementInfo` are not intended to be in this PR?

Reply:
Yes, I revert these changes